broke-broker/src/main/java/org/dsngroup/broke/broker/processor/MqttMessageHandler.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
```

Comment:
I think the `handler` can move to `broker/handler` package.

Reply:
OK

Reply:
I think handler can move to ```broker/handler``` package.
And I think it's better to move ```broker/processor``` to ```broker/handler/processor``` because all of the logics in ```broker/processor``` are currently used by the ```handler```

Reply:
Ok, sounds good!